scripts/scripts.js

```diff
@@ -178,6 +178,17 @@ function preloadFile(href, as) {
   document.head.appendChild(link);
 }
 
+function listenForLCPEvent() {
+  const observer = new PerformanceObserver((list) => {
+    const entries = list.getEntries();
+    if (entries.length > 0) {
+      events.emit('eds/lcp', true);
+    }
+  });
+
+  observer.observe({ type: 'largest-contentful-paint', buffered: true });
+}
+
```

Comment:
[LargestContentfulPaint](https://developer.mozilla.org/en-US/docs/Web/API/LargestContentfulPaint) is not supported on Safari, you cannot use the PerformanceObserver here.

Reply:
I'll change the approach.